algolia/search/requests_indexing.go

```diff
@@ -42,3 +42,8 @@ type getObjectsReq struct {
 type batchReq struct {
 	Requests []BatchOperation `json:"requests,omitempty"`
 }
+
+type PartialUpdateOperation struct {
```

Comment:
Can you explain the context buddy?

Reply:
This related to the documentation change which requires to add snippets for partial update operations, such as, in Go:

```go
    type Record struct {
        ObjectID string                        `json:"objectID"`
        Count    search.PartialUpdateOperation `json:"count"`
    }
    res, err := index.PartialUpdateObjects([]Record{
        {
            ObjectID: "myID",
            Count: search.PartialUpdateOperation{
                Operation: "Increment",
                Value:     2,
            },
        },
    })
```